### PR TITLE
Version 2 of Clipper contracts for ETH Mainnet and Polygon for `dex.trades`

### PR DESCRIPTION
**Pull request by aothman**
Clipper recently replaced its contracts on Mainnet and Polygon. This PR updates the `dex.trades` table on each network with the new contracts, whose trades will be as version "2" of the project "Clipper". The specific function implementation is called `insert_clipper_v2` on both networks.

I've checked that:

* [ X ] I tested the relevant subquery on dune.com
* [ N/A ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ X ] the directory tree matches the pattern /sector/blockchain/ e.g. /tokens/ethereum
* [ N/A ] if adding a new model, I added a test
* [ X ] the filename is unique and ends with .sql
* [ X ] each file has only one view, table or function defined  
* [ X ] column names are `lowercase_snake_cased`

### CODE REVIEW

File: ethereum/dex/trades/insert_clipper_v2.sql [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        tx_hash,
+        tx_from,
+        tx_to,
+        trace_address,
+        evt_index,
+        trade_id
+    )
+    SELECT
+        dexs.block_time,
+        pa.symbol AS token_a_symbol,
```
Comment (jeff-dude) [inline ethereum/dex/trades/insert_clipper_v2.sql:32]: for lines 32-35, you pull this data from prices, but in v1 script, it pulls from erc20 tokens table. do you want to keep consistency? i think it'll be more reliable too, as prices table can fall behind as it depends on data from outside API.

File: ethereum/dex/trades/insert_clipper_v2.sql [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    FROM (
+        -- Clipper v2 "Caravel" on ETH Mainnet
+        SELECT
+            t.evt_block_time AS block_time,
+            'Clipper' AS project,
+            '2' AS version,
+            'DEX' AS category,
+            t."recipient" AS trader_a,
+            NULL::bytea AS trader_b,
+            "inAmount" AS token_b_amount_raw,
```
Comment (jeff-dude) [inline ethereum/dex/trades/insert_clipper_v2.sql:66]: in v1 script, token_a = "inAmount" while this v2 script has token_b = "inAmount" -- do they need flipped here?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: ethereum/dex/trades/insert_clipper_v2.sql (hunk 1 above)
Comment (aothman) [reply ethereum/dex/trades/insert_clipper_v2.sql]: Updated to pull symbol and decimals data from the erc20 tokens table.

File: ethereum/dex/trades/insert_clipper_v2.sql (hunk 2 above)
Comment (jeff-dude) [reply ethereum/dex/trades/insert_clipper_v2.sql:66]: a change to v1 would require a reload, yes. that is up to you. are there downstream queries/dashboards which have a manual workaround on the flipped values, that would also need updated if we change?

File: ethereum/dex/trades/insert_clipper_v2.sql [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    AND project = 'Clipper'
+    AND version = '2'
+);
+
+INSERT INTO cron.job (schedule, command)
+VALUES ('*/10 * * * *', $$
+    SELECT dex.insert_clipper_v2(
+        (SELECT max(block_time) - interval '1 days' FROM dex.trades WHERE project='Clipper' AND version='2'),
+        (SELECT now() - interval '20 minutes'),
+        (SELECT max(number) FROM ethereum.blocks WHERE time < (SELECT max(block_time) - interval '1 days' FROM dex.trades WHERE project='Clipper')),
```
Comment (jeff-dude) [inline ethereum/dex/trades/insert_clipper_v2.sql:125]: i don't have access, but i pushed a change locally and added the version filter from line 123 to this spot as well. please add as a commit prior to merge

File: polygon/dex/trades/insert_clipper_v2.sql [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        tx_hash,
+        tx_from,
+        tx_to,
+        trace_address,
+        evt_index,
+        trade_id
+    )
+    SELECT
+        dexs.block_time,
+        pa.symbol AS token_a_symbol,
```
Comment (jeff-dude) [inline polygon/dex/trades/insert_clipper_v2.sql:32]: did you want to change these similar to the ethereum script? i can change locally and run, but then would need your access to push commit here

File: ethereum/dex/trades/insert_clipper_v2.sql (hunk 2 above)
Comment (aothman) [reply ethereum/dex/trades/insert_clipper_v2.sql:66]: Let's leave it alone then. Not hurting anything and it's no longer active.

**Review by jeff-dude — APPROVED** [review]
@aothman thank you! should be all set

**Pull request merged**